apps/gbridge/widget.js

```diff
@@ -192,10 +196,24 @@
     g.flip(); // turns screen on
   }
 
-  NRF.on("connect", changedConnectionState);
-  NRF.on("disconnect", changedConnectionState);
+  function reload() {
+    settings = storage.readJSON('gbridge.json', 1) || {};
+    if (!("showIcon" in settings)) {
+      settings.showIcon = true;
+    }
+    if (settings.showIcon) {
+      WIDGETS["gbridgew"].width = 24;
+      NRF.on("connect", changedConnectionState);
```

Comment:
If you happen to call 'reload' more than once without changing `settings.showIcon`, I think this'll just queue up multiple event listeners? Maybe add `NRF.removeListener` first to run in all cases?